app/qml/FormController.qml

```diff
@@ -0,0 +1,143 @@
+﻿/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import "."
+import "./form" as Forms
+import QgsQuick 0.1 as QgsQuick
+import lc 1.0
+
+Item {
+  id: root
+
+  /*
+   * FeaturePanel component is responsible for entire feature form, both preview and fullscreen form
```

Comment:
FeaturePanel? do you mean FormController? 

the name is a bit consusing to me at the first read thinking what is the difference between form controller and attribute controller? Maybe `FormsStackManager`? 

it would be good to add here descrition, e.g. last form is not destroyed but reused, new forms pages are opened e.g. when using relation in forms, ...